AUTHORS.txt

```diff
@@ -533,6 +533,7 @@ Liu Ya <liuya54892@gmail.com>
 Liu Yang <zation1@gmail.com>
 LongYinan <lynweklm@gmail.com>
 Loogeek <340017792@qq.com>
+Rinku Chaudhari <github@therealrinku.com>
```

Comment:
This file is automatically generated by the script, does not require manual modification.